docs/reference/cluster/health.asciidoc

```diff
@@ -1,18 +1,44 @@
 [[cluster-health]]
 === Cluster Health
 
```

Comment:
Per the [API Reference template](https://raw.githubusercontent.com/elastic/docs/master/shared/api-ref-ex.asciidoc), we'll typically use a short description here.
```suggestion
Returns the health status of a cluster
```